tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java

```diff
@@ -44,6 +44,13 @@
   int GcDeleteRangeMaxBackoff = 100000;
   int rawkvMaxBackoff = 40000;
   int splitRegionBackoff = 20000;
+  int commitMaxBackoff = 3000;
+
+  /** 20 seconds */
+  int batchPrewriteBackoff = 20000;
```

Comment:
final int seconds = 1000;
int batchPrewriteBackoff =20*seconds;
int batchCommitBackoff = 3 * seconds;

Above code does not need comments. What do you think? We can totally fix this later. It depends on you.

Reply:
I prefer to fix later